src/dbwrapper.cpp

```diff
@@ -155,11 +156,28 @@ CDBWrapper::~CDBWrapper()
 
 bool CDBWrapper::WriteBatch(CDBBatch& batch, bool fSync)
 {
+    const bool log_memory = LogAcceptCategory(BCLog::LEVELDB);
+    double mem_before;
```

Comment:
Nit, initialize `= 0`.

Reply:
done (although I think it's fine to keep unused variables uninitialized)